processors/project-status-labeler.ts

```diff
@@ -0,0 +1,109 @@
+import moment = require('moment')
+import {GitHubClient} from '../github'
+import {CrawlingTarget} from '../interfaces'
+import {getLastCommentField, IssueList, ProjectIssue} from '../project-reports-lib'
+
+const now = moment()
+
+const reportType = 'project'
+export {reportType}
+
+/**
+ * Reads the latest status from a comment and creates the corresponding label
+ */
+export function getDefaultConfiguration(): any {
```

Comment:
I'm curious if we can get away from these `any` types, since they remove the benefits of TypeScript. Does just returning a non-any type break references elsewhere? If so, we could try and come up with a type for configuration that's a bit more specific.

Reply:
sure.  I'll make that change.